### PR TITLE
Update proteomicslfq.rst

### PR DESCRIPTION
**Pull request by timosachsenberg**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/quantms/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/quantms _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/proteomicslfq.rst [hunk 1]
```diff
@@ -1,2 +1,18 @@
...
+
+1. **ID-based feature extraction** uses targeted feature dectection using RT and m/z information derived from identification data to extract features. Only identifications found in a particular MS run are used to extract features in the same run. No transfer of IDs (match between runs) is performed.
+2. **ID-based + untargeted extraction** adds untargeted feature detection to obtain quantities from unidentified features. Transfer of Ids (match between runs) is performed by transfering feature identifications to coeluting, unidentified features with similar mass and RT in other runs.
+         
+Requantification:
+1. Optionally, a requantification step is performed that tries to fill NA values. If a peptide has been quantified in more than half of all maps, the peptide is selected for requantification. In that case, the mean observed RT (and theoretical m/z) of the peptide is used to perform a second round of targeted extraction.
+
+Output:
+- mzTab file with analysis results
+- Aalysis results for statistical downstream analysis in MSstats and Triqler
```
Comment (jpfeuffer) [inline docs/proteomicslfq.rst:17]:
```suggestion
- Analysis results for statistical downstream analysis in MSstats and Triqler
```

File: docs/proteomicslfq.rst [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+The ProteomicsLFQ tool performs label-free quantification of peptides and proteins.
+
+Feature extraction:
+ProteomicsLFQ supports **ID-based feature extraction** or a combined **ID-based + untargeted extraction**.
+
+1. **ID-based feature extraction** uses targeted feature dectection using RT and m/z information derived from identification data to extract features. Only identifications found in a particular MS run are used to extract features in the same run. No transfer of IDs (match between runs) is performed.
+2. **ID-based + untargeted extraction** adds untargeted feature detection to obtain quantities from unidentified features. Transfer of Ids (match between runs) is performed by transfering feature identifications to coeluting, unidentified features with similar mass and RT in other runs.
+         
+Requantification:
+1. Optionally, a requantification step is performed that tries to fill NA values. If a peptide has been quantified in more than half of all maps, the peptide is selected for requantification. In that case, the mean observed RT (and theoretical m/z) of the peptide is used to perform a second round of targeted extraction.
```
Comment (jpfeuffer) [inline docs/proteomicslfq.rst:13]:
```suggestion
1. Optionally, a requantification step is performed that tries to fill missing values. If a peptide has been quantified in more than half of all maps, the peptide is selected for requantification. In that case, the mean observed RT (and theoretical m/z) of the peptide is used to perform a second round of targeted extraction.
```

File: docs/proteomicslfq.rst [hunk 3]
```diff
@@ -1,2 +1,18 @@
 ProteomicsLFQ metatool (OpenMS)
 ===============================
+
+The ProteomicsLFQ tool performs label-free quantification of peptides and proteins.
+
+Feature extraction:
+ProteomicsLFQ supports **ID-based feature extraction** or a combined **ID-based + untargeted extraction**.
+
+1. **ID-based feature extraction** uses targeted feature dectection using RT and m/z information derived from identification data to extract features. Only identifications found in a particular MS run are used to extract features in the same run. No transfer of IDs (match between runs) is performed.
+2. **ID-based + untargeted extraction** adds untargeted feature detection to obtain quantities from unidentified features. Transfer of Ids (match between runs) is performed by transfering feature identifications to coeluting, unidentified features with similar mass and RT in other runs.
```
Comment (jpfeuffer) [inline docs/proteomicslfq.rst:10]:
```suggestion
2. **ID-based + untargeted extraction** adds untargeted feature detection to obtain quantities from unidentified features. Transfer of IDs (match between runs) is performed by transfering feature identifications to coeluting, unidentified features with similar mass and RT in other runs.
```

File: docs/proteomicslfq.rst [hunk 4]
```diff
@@ -1,2 +1,18 @@
 ProteomicsLFQ metatool (OpenMS)
 ===============================
+
+The ProteomicsLFQ tool performs label-free quantification of peptides and proteins.
+
+Feature extraction:
+ProteomicsLFQ supports **ID-based feature extraction** or a combined **ID-based + untargeted extraction**.
+
+1. **ID-based feature extraction** uses targeted feature dectection using RT and m/z information derived from identification data to extract features. Only identifications found in a particular MS run are used to extract features in the same run. No transfer of IDs (match between runs) is performed.
```
Comment (jpfeuffer) [inline docs/proteomicslfq.rst:9]:
```suggestion
1. **ID-based feature extraction** uses targeted feature detection using RT and m/z information derived from identification data to extract features. Only identifications found in a particular MS run are used to extract features in the same run. No transfer of IDs (match between runs) is performed.
```

**Review by jpfeuffer — CHANGES_REQUESTED** [review]
Nice. But see suggestions. At some point we should add references to or at least mention the actual parameter values.
RT alignment is a bit missing here. I think linking is pretty self explanatory but could get a sentence, too.
Regarding ID, maybe add that it only takes IDs passing the FDR threshold and best PSM per spectrum?

**Comment by jpfeuffer** [comment]
Yes for example ProteomicsLFQ has inference included, but I will be making a separate chapter since it is used in isobaric as well

**Comment by timosachsenberg** [comment]
How do we do references to the corresponding OpenMS documentation?

**Comment by jpfeuffer** [comment]
Just links, or what do you mean? Whenever speaking about an OpenMS tool I would cite the paper if available and probably link the docs, yes.

**Comment by timosachsenberg** [comment]
yes should we use latest or a specific version (that needs to be updated) https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/release/latest/html/index.html

**Comment by jpfeuffer** [comment]
If you find a way to specify the version once with a variable, then this.
Otherwise latest

**Comment by jpfeuffer** [comment]
https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#substitution-definitions

**Pull request merged**